src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -0,0 +1,211 @@
+        values={{ modelName: data.modelPlan.modelName }}
+      />
+    );
+  }
+  if (isTaggedInDiscussionReply(data)) {
+    return (
+      <Trans
+        i18nKey="notifications:index.activityType.taggedInDiscussionReply.text"
+        values={{ modelName: data.modelPlan.modelName }}
+      />
+    );
+  }
+  if (isDailyDigest(data)) {
+    return (
+      <Trans i18nKey="notifications:index.activityType.dailyDigestComplete.text" />
+    );
+  }
+  return '';
+};
+
+export const ActivityCTA = ({
+  data,
+  isExpanded
+}: {
+  data: MetaDataType;
+  isExpanded: boolean;
+}) => {
+  if (isTaggedInDiscussion(data)) {
+    return (
+      <>
+        <Trans i18nKey="notifications:index.activityType.taggedInDiscussion.cta" />
+        <Icon.ArrowForward className="margin-left-1" aria-hidden />
+      </>
+    );
+  }
+  if (isTaggedInDiscussionReply(data)) {
+    return (
+      <>
+        <Trans i18nKey="notifications:index.activityType.taggedInDiscussionReply.cta" />
+        <Icon.ArrowForward className="margin-left-1" aria-hidden />
+      </>
+    );
+  }
+  if (isDailyDigest(data)) {
+    return (
+      <>
+        <Trans i18nKey="notifications:index.activityType.dailyDigestComplete.cta" />
+        {isExpanded ? (
+          <Icon.ExpandLess className="margin-left-1" aria-hidden />
+        ) : (
+          <Icon.ExpandMore className="margin-left-1" aria-hidden />
+        )}
+      </>
+    );
+  }
+  return <></>;
+};
+export const TranslateStatusChange = ({
+  status
+}: {
+  status: string | null;
+}) => {
+  switch (status) {
+    case 'PLAN_DRAFT':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.plan_draft" />
+      );
+    case 'PLAN_COMPLETE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.plan_complete" />
+      );
+    case 'ICIP_COMPLETE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.icip_complete" />
+      );
+    case 'INTERNAL_CMMI_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.internal_cmmi_clearance" />
+      );
+    case 'CMS_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.cms_clearance" />
+      );
+    case 'HHS_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.hhs_clearance" />
+      );
+    case 'OMB_ASRF_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.omb_asrf_clearance" />
+      );
+    case 'CLEARED':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.cleared" />
+      );
+    case 'ANNOUNCED':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.announced" />
+      );
+    default:
+      return <></>;
+  }
+};
+
+export const translatePlanSections = (section: string) => {
```

Comment:
Again we don't really need this util.  The section is defined as a string, but we should have that more strongly typed.  IIRC we have a gql type for the plan sections.  This can all be automated by correct translation mappings